features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/TreeViewTraversal.kt

```diff
@@ -86,7 +91,7 @@ internal class TreeViewTraversal(
         val resolvedWireframes = internalLogger.measureMethodCallPerf(
             javaClass,
             "$METHOD_CALL_MAP_PREFIX ${mapper.javaClass.simpleName}",
-            METHOD_CALL_SAMPLING_RATE
+            getSamplingRateForMapper(mapper.javaClass.simpleName)
```

Comment:
> [!WARNING]
> Using a different sampling rate per mapper means that it'll skew our estimation of which ones are more or less used. Given that those mappers will be called a lot anyway, I'd be in favor of using the lowest sampling rate for all of them, to be able to track also which ones are more used throughout our customers' apps.

Reply:
If we don't need different sample rates per mapper then the whole pr becomes much simpler. We need only to change the sample rate for all of "Map With" telemetry to 0.001f

Reply:
👍  I've changed to have all mappers use the lowest rate